src/scenes/ProgressReports/Detail/ProductTable.tsx

```diff
@@ -20,7 +30,15 @@ type RowData = {
   [K in ProductStep]?: string;
 };
 
-export const ProductTable = ({ products, category }: ProductTableProps) => {
+export const ProductTable = ({
+  products,
+  category,
+  editable,
+  editMode = 'row',
+  hidePagination = true,
```

Comment:
There was a reason I made the negation. The `pagination` on the Grid is only `true` or `undefined`, it can't be false:
```ts
/**
 * The props users can give to the `DataGrid` component.
 */
export declare type DataGridProps<R extends GridValidRowModel = any> = Omit<Partial<DataGridPropsWithDefaultValues> & DataGridPropsWithComplexDefaultValueBeforeProcessing & DataGridPropsWithoutDefaultValue<R>, DataGridForcedPropsKey> & {
    pagination?: true;
};
```

Reply:
I ended up making that change